server/data/converters/modelTypes.ts

```diff
@@ -24,3 +24,66 @@ export type ServiceArea = {
   slug: string
   products: Product[]
 }
+
+export type Summary = {
+  config?: Record<string, number>
+  secret?: Record<string, number>
+  'os-pkgs'?: {
+    fixed?: Record<string, number>
+    unfixed?: Record<string, number>
+  }
+  'lang-pkgs'?: {
+    fixed?: Record<string, number>
+    unfixed?: Record<string, number>
+  }
+}
+
+export type ScanResult = {
+  config?: Array<{
+    FilePath: string
+    Severity: string
+    LineNumber: string
+    Description: string
+    AdditionalContext: string
+  }>
+  secret?: Array<{
+    Severity: string
+    FilePath: string
+    LineNumber: string
+    Description: string
+    AdditionalContext: string
+  }>
+  'os-pkgs'?: Array<{
+    PkgName: string
+    Severity: string
+    Description: string
+    InstalledVersion: string
+    FixedVersion: string
+    VulnerabilityID: string
+    PrimaryURL: string
+  }>
+  'lang-pkgs'?: Array<{
+    PkgName: string
+    Severity: string
+    Description: string
+    InstalledVersion: string
+    FixedVersion: string
+    VulnerabilityID: string
+    PrimaryURL: string
+  }>
+}
+
+export type ScanSummary = {
+  summary?: Summary
+  scan_result?: ScanResult
+}
+
+export type trivyScanType = {
```

Comment:
Only suggestion is call it `TrivyScanType` as we capitalise types and associated change in `trivyScans.ts`. Otherwise all good.